src/pint_pal/utils.py

```diff
@@ -716,7 +716,7 @@ def alert(s):
             and pm.frozen
             and pm.value is not None
             and pm.value != 0):
-            if p in {"START", "FINISH", "POSEPOCH", "DMEPOCH", "PEPOCH", "TZRMJD", "DM", "DMX", "NTOA", "CHI2", "DMDATA", "TZRFRQ", "RNAMP", "RNIDX"}:
+            if p in {"START", "FINISH", "POSEPOCH", "DMEPOCH", "PEPOCH", "TZRMJD", "DM", "DMX", "NTOA", "CHI2", "DMDATA", "TZRFRQ", "RNAMP", "RNIDX", "CHI2R", "TRES", "SWP"}:
```

Comment:
To clarify: "CHI2R", "TRES", and "SWP" are parameters it doesn't make sense to fit anyway, right?